pkg/openshift/certgen/release39/templates/master/etc/etcd/etcd.conf

```diff
@@ -48,6 +48,7 @@ ETCD_PEER_KEY_FILE=/etc/etcd/peer.key
 
 #[logging]
 ETCD_DEBUG="False"
```

Comment:
`ETCD_DEBUG="True"` is too chatty because of the gossip protocol so I didn't enable it here - we may want to use it eventually if bumping the etcdserver loglevel is not enough.